javaassignment.js

```diff
@@ -0,0 +1,381 @@
+/*question # 1*/
+function ph(i){
+    if(i>7){
+        return 'acidic';
+    }else if (i<7){
+        return 'basic';
+    }else if (i=7) {
+        return 'neutral';
+    } else {
+        return 'undefined';
+    }
+ }
+ /*
+ console.log('the out put for 7 is  ' + ph(7))
+ console.log('the out put for 6 is  ' + ph(6))
+ console.log('the out put for 3 is  ' + ph(3));
+PS C:\Users\bisrat\Desktop\source> node test.js
+the out put for 7 is  neutral
+the out put for 6 is  basic
+the out put for 3 is  basic 
+
+*****************************************************************
+
+ question #2 */
+
+function dis(price){
+ if(price >=300){
+    price=price-(price*(50/100));
+    console.log(price);
+}else if (200<price<300){
+    price=price-(price*(20/100));
+    console.log(price);
+}else if (0<price<=200){
+    price=price-(price*(5/100));
+    console.log(price); 
+}else
+{
+    console.log('error');
+}
+ }
+ 
+ console.log(dis(300));
+/*
+PS C:\Users\bisrat\Desktop\source> node assignement.js
+150
+   */
+/***************************************************************************************
+
+/*question #3 */
+function shi(price, location){
+    if(location == 'EU')
+    {
+        let totalcost ;

```

Comment:
you could declare let totalCost outside of the if statement so that you don't have to repeat it inside the else-ifs below